crates/pixi_git/src/lib.rs

```diff
@@ -15,6 +15,13 @@ pub mod url;
 /// The query parameter used to specify the type of reference in a Git URL.
 pub const GIT_URL_QUERY_REV_TYPE: &str = "rev_type";
 
+/// The warning message used in reporter for SSH cloning.
+/// This is intended to help user understand that they need to set their SSH passphrase
+/// before cloning a repository using SSH, otherwise the process can hang.
+/// Original issue: <https://github.com/prefix-dev/pixi/issues/3709>
+pub const GIT_SSH_CLONING_WARNING_MSG: &str =
+    "Ensure your passphrase is set with ssh-add beforehand to avoid cloning issues.";
```

Comment:
It's only a warning if there is an issue, let's not confuse the other people that just have to wait.
```suggestion
    "Heads-up: use `ssh-add` if this hangs.";
```